hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/table/format/TestInputFormat.java

```diff
@@ -1379,6 +1379,34 @@ void testStreamWriteAndReadWithUpgrade(HoodieTableType tableType) throws Excepti
     TestData.assertRowDataEquals(result, TestData.DATA_SET_INSERT);
   }
 
+  @Test
+  public void testWriteCowWithPartialUpdate() throws Exception {
```

Comment:
It's great if we can also add a test case in `TestPartialUpdateFlinkRecordMerger`.